### DISCUSSION TITLE
Has anyone exposed Gotenberg via an HTTPS endpoint on AWS and happy to share their implementation ?

### DISCUSSION THREAD

**Discussion by haydenpitout** [gotenberg/gotenberg#672] [Q&A]
I've taken note of two topics touching on this subject:

https://github.com/gotenberg/gotenberg/discussions/502
https://github.com/gotenberg/gotenberg/issues/540

I'm keen to find out if anyone has exposed Gotenberg via an HTTPS endpoint (API Gateway?) on AWS and is happy to share their implementation ... even just the key files use and any possible gotchas encountered along the way.

I'm keen to set Gotenberg up as a stand alone service that our frontend (not hosted on AWS) can call.

Any help would be greatly appreciated.

**Comment by gulien** [comment]
By default, I’ll advise against exposing Gotenberg to the external world

That being said, there are other alternatives that are easier to manage than AWS if the rest of your stack is not already in AWS.

Please checkout Render for instance, it manages all the pain points for you (and provides an HTTPS endpoint by default).

The demo instance runs smoothly on Render for example.

**Reply by haydenpitout** [reply]
Thanks @gulien ... i'll look into Render as an option  :-)
